pyqtgraph/graphicsItems/PlotDataItem.py

```diff
@@ -206,102 +206,127 @@ def applyLogMapping(self, logMode: tuple[bool, bool]):
 
 class PlotDataItem(GraphicsObject):
     """
-    Provides a unified interface for displaying plot curves, scatter plots, or both.
-
-    The symbols of a scatter point are rendered over the "point" of the data, and the 
-    curve connects adjacent points.
+    PlotDataItem is PyQtGraph's primary way to plot 2D data. It provides a unified 
```

Comment:
First line in the docstring needs to be confined to 1-line per numpydoc:

https://numpydoc.readthedocs.io/en/latest/format.html#short-summary